### DISCUSSION TITLE
How to get the "Pair Extraordinaire" achievement? I don't quite understand

### DISCUSSION THREAD

**Discussion by westinyang** [Schweinepriester/github-profile-achievements#176] [Q&A] [answered]
How to get the "Pair Extraordinaire" achievement? I don't quite understand

**Comment by Chameleon-Coder** [comment, marked as answer]
As @thenithinbalaji said:
> Make a PR with people as your coauthors.
> 
> Tell someone to make a PR with you as a coauthor. Here is how you can add a coauthor, https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

**Reply by thenithinbalaji** [reply]
Yes, please take a look at this full discussion   
https://github.com/Schweinepriester/github-profile-achievements/discussions/127

**Comment by haideralicodes** [comment]
This is one way to do it

create a branch and commit changes
get a collaborator to commit changes on the same branch
create a pull request
merge to default branch using squash and merge

**Comment by thenithinbalaji** [comment]
Make a PR with people as your co-authors.

Tell someone to make a PR with you as a co-author. 
Here is how you can add a coauthor,
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

Take a look at the comment thread of 127 discussion where we discussed getting this badge https://github.com/Schweinepriester/github-profile-achievements/discussions/127
